### PR TITLE
Various text updates

### PR DESCRIPTION
**Pull request by zmarotrix**
Changed "Randomiser" to "Randomizer" for consistency with the rest of the Zelda community, deleted the broken byte swapper and updated the credits in "About". Also added version number to the window title.

### CODE REVIEW

File: fAbout.Designer.cs [hunk 1]
```diff
@@ -28,49 +28,66 @@ protected override void Dispose(bool disposing)
...
-            this.Icon = ((System.Drawing.Icon)(resources.GetObject("$this.Icon")));
-            this.MaximizeBox = false;
-            this.Name = "fAbout";
-            this.Text = "About";
-            this.ResumeLayout(false);
-            this.PerformLayout();
-
+            System.ComponentModel.ComponentResourceManager resources = new System.ComponentModel.ComponentResourceManager(typeof(fAbout));
+            this.lAboutText = new System.Windows.Forms.Label();
+            this.label1 = new System.Windows.Forms.Label();
```
Comment (ZoeyZolotova) [inline fAbout.Designer.cs]: Please give label1 and linkLabel1 meaningful names.

File: fAbout.Designer.cs [hunk 2]
```diff
@@ -28,49 +28,66 @@ protected override void Dispose(bool disposing)
...
+            this.lAboutText.Click += new System.EventHandler(this.lAboutText_Click);
+            // 
+            // label1
+            // 
+            this.label1.Font = new System.Drawing.Font("Microsoft Sans Serif", 16F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
+            this.label1.Location = new System.Drawing.Point(0, 6);
+            this.label1.Name = "label1";
+            this.label1.Size = new System.Drawing.Size(359, 27);
+            this.label1.TabIndex = 1;
+            this.label1.Text = "Majoras Mask Randomizer";
```
Comment (ZoeyZolotova) [inline fAbout.Designer.cs]: Should it be `Majora's` not `Majoras`?

File: fAbout.cs [hunk 3]
```diff
@@ -15,7 +15,6 @@ public partial class fAbout : Form
         public fAbout()
         {
             InitializeComponent();
-            lVer.Text = AssemblyVersion;
```
Comment (ZoeyZolotova) [inline fAbout.cs]: Why is this being removed?

File: fAbout.cs [hunk 4]
```diff
@@ -25,6 +24,16 @@ public string AssemblyVersion
                 Version v = Assembly.GetExecutingAssembly().GetName().Version;
                 return String.Format("v{0}.{1}", v.Major, v.Minor);
             }
-        }
+        }
+
+        private void lAboutText_Click(object sender, EventArgs e)
```
Comment (ZoeyZolotova) [inline fAbout.cs]: Looks unnecessary, can you remove it?

File: fAbout.resx [hunk 5]
```diff
@@ -1,2969 +1,2969 @@
...
-        Zf+1mGr/tZpv/7Wbcv+1nHL/tZxz/7Wcc/+1m3L/tZxz/7Wccv+1nHL/tJtx/7SSYZGyfj0GAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAArnkgAqSgSw6duGhEl817vZbVg/eT3Yv/k96L/5Tdi/2aw2+5o6FKEgAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACueSAOmsJwlZLc
-        iv+Q2oj/jteF/43Vg/+L1IL/i9F+/467aduSplBslJxIEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAA
```
Comment (ZoeyZolotova) [inline fAbout.resx]: `Majora's`

File: fAbout.resx [hunk 6]
```diff
@@ -1,2969 +1,2969 @@
...
-  <data name="lAboutText.Text" xml:space="preserve">
-    <value>Majora's Mask Randomiser (External)
-
-An item randomiser for The Legend of Zelda: Majora's Mask ROMs.
-
-Research and programming by DeathBasket
-
-Thanks:
-Jimmie1717 - random stuff
-KiKe - wadpacker</value>
```
Comment (ZoeyZolotova) [inline fAbout.resx]: Probably want to keep these credits in here.

File: fMain.Designer.cs [hunk 7]
```diff
@@ -30,957 +30,891 @@ protected override void Dispose(bool disposing)
...
+            this.cMixSongs.AutoSize = true;
+            this.cMixSongs.BackColor = System.Drawing.Color.Transparent;
+            this.cMixSongs.CheckAlign = System.Drawing.ContentAlignment.MiddleRight;
+            this.cMixSongs.Font = new System.Drawing.Font("Microsoft Sans Serif", 8.25F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
+            this.cMixSongs.ForeColor = System.Drawing.Color.Black;
+            this.cMixSongs.Location = new System.Drawing.Point(22, 22);
+            this.cMixSongs.Name = "cMixSongs";
+            this.cMixSongs.Size = new System.Drawing.Size(122, 17);
+            this.cMixSongs.TabIndex = 3;
+            this.cMixSongs.Text = "Mix son
```
Comment (ZoeyZolotova) [inline fMain.Designer.cs]: Please don't hard-code the version name into this text. Don't want to have to modify the designer file every time we do a release.

File: fAbout.cs (hunk 3 above)
Comment (zmarotrix) [reply fAbout.cs]: It no longer exists.

File: fAbout.cs (hunk 4 above)
Comment (zmarotrix) [reply fAbout.cs]: The blank space? yeah, I'll do that.

File: fMain.Designer.cs (hunk 7 above)
Comment (zmarotrix) [reply fMain.Designer.cs]: I'll try to find a way to add it without hardcoding.

File: fAbout.Designer.cs (hunk 1 above)
Comment (zmarotrix) [reply fAbout.Designer.cs]: It was automatically added by the designer. I'll go back and rename it.

File: fAbout.Designer.cs (hunk 2 above)
Comment (zmarotrix) [reply fAbout.Designer.cs]: You are correct. I'll update this.

File: fAbout.Designer.cs [hunk 8]
```diff
@@ -28,49 +28,67 @@ protected override void Dispose(bool disposing)
...
+            // 
+            // discordLabelText
+            // 
+            this.discordLabelText.Location = new System.Drawing.Point(27, 33);
+            this.discordLabelText.Name = "discordLabelText";
+            this.discordLabelText.Size = new System.Drawing.Size(308, 145);
+            this.discordLabelText.TabIndex = 0;
+            this.discordLabelText.Text = resources.GetString("discordLabelText.Text");
+            this.discordLabelText.TextAlign = System.Drawing.ContentAlignment.MiddleCenter;
+            this.discordLabelText.Click += new System.EventHandler(this.discordLabelText_Click);
```
Comment (ZoeyZolotova) [inline fAbout.Designer.cs]: Please remove this unused click event listener and the `discordLabelText_Click` function.

File: fAbout.Designer.cs [hunk 9]
```diff
@@ -28,49 +28,67 @@ protected override void Dispose(bool disposing)
...
+            // discordLabel
+            // 
+            this.discordLabel.Font = new System.Drawing.Font("Microsoft Sans Serif", 16F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
+            this.discordLabel.Location = new System.Drawing.Point(0, 1);
+            this.discordLabel.Name = "discordLabel";
+            this.discordLabel.Size = new System.Drawing.Size(359, 27);
+            this.discordLabel.TabIndex = 1;
+            this.discordLabel.Text = "Majora\'s Mask Randomizer";
+            this.discordLabel.TextAlign = System.Drawing.ContentAlignment.MiddleCenter;
+            this.discordLabel.Click += new System.EventHandler(this.label1_Click);
```
Comment (ZoeyZolotova) [inline fAbout.Designer.cs]: Please remove this unused click event listener and the `label1_Click` function.

File: fAbout.Designer.cs [hunk 10]
```diff
@@ -28,49 +28,67 @@ protected override void Dispose(bool disposing)
...
+            // 
+            this.discordLinkLabel.AutoSize = true;
+            this.discordLinkLabel.Location = new System.Drawing.Point(108, 119);
+            this.discordLinkLabel.Name = "discordLinkLabel";
+            this.discordLinkLabel.Size = new System.Drawing.Size(141, 13);
+            this.discordLinkLabel.TabIndex = 2;
+            this.discordLinkLabel.TabStop = true;
+            this.discordLinkLabel.Text = "https://discord.gg/8qbreUM";
+            this.discordLinkLabel.TextAlign = System.Drawing.ContentAlignment.MiddleCenter;
+            this.discordLinkLabel.LinkClicked += new System.Windows.Forms.LinkLabelLinkClickedEventHandler(this.linkLabel1_LinkClicked);
```
Comment (ZoeyZolotova) [inline fAbout.Designer.cs]: Please update the name of the `linkLabel1_LinkClicked` function to `discordLinkLabel_LinkClicked`

File: fAbout.resx [hunk 11]
```diff
@@ -1,2969 +1,2970 @@
...
-        Zf+1mGr/tZpv/7Wbcv+1nHL/tZxz/7Wcc/+1m3L/tZxz/7Wccv+1nHL/tJtx/7SSYZGyfj0GAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAArnkgAqSgSw6duGhEl817vZbVg/eT3Yv/k96L/5Tdi/2aw2+5o6FKEgAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACueSAOmsJwlZLc
-        iv+Q2oj/jteF/43Vg/+L1IL/i9F+/467aduSplBslJxIEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAA
```
Comment (ZoeyZolotova) [inline fAbout.resx]: Please revert the renaming of this resource.

File: fAbout.resx (hunk 5 above)
Comment (ZoeyZolotova) [reply fAbout.resx]: Please update "Maintained by the Majoras Mask Randomizer Community Discord." Majoras to Majora's

File: fMain.Designer.cs (hunk 7 above)
Comment (ZoeyZolotova) [inline fMain.Designer.cs]: Please remove the commented out code.

File: fAbout.resx (hunk 11 above)
Comment (ZoeyZolotova) [reply fAbout.resx]: Referring to changing it back to `lAboutText`

File: fAbout.cs [hunk 12]
```diff
@@ -15,16 +15,21 @@ public partial class fAbout : Form
...
-            }
-        }
+        }
+
+        private void discordLinkLabel_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
+        {
+            System.Diagnostics.Process.Start("https://discord.gg/8qbreUM");
+        }
+
+        private void label1_Click(object sender, EventArgs e)
```
Comment (ZoeyZolotova) [inline fAbout.cs]: Please remove these two empty functions.

**Pull request merged**